README.md

```diff
@@ -18,6 +20,14 @@ You can try out the user-side of Calypso on [WordPress.com](https://wordpress.co
 4.	Execute `make run` or `make dashboard` (for a more visually-oriented interface) from the root directory of the repository.
 5.	Open `calypso.localhost:3000` in your browser.
 
+### Docker
+
+1. Download [Docker for Mac](https://docs.docker.com/docker-for-mac/), [Docker for Windows](https://docs.docker.com/docker-for-windows/) unless you are in linux, in which case, just install [Docker](https://docs.docker.com/engine/installation/linux/)
```

Comment:
It's probably fine to simply link to the Docker installation page as it lists the variations of operating systems on which it will run plus installation instructions.